zerver/lib/actions.py

```diff
@@ -817,6 +873,7 @@ def check_send_message(*args, **kwargs):
     return do_send_messages([message])[0]
```

Comment:
I haven't been able to find out how to impose same argspec as `check_send_message`. I'll put `(*Any, **Any) -> int` for now.